<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTab</class>
 <widget class="QWidget" name="SetupTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbTcpIP">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>Tcp/IP</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="loTcpIpGroup">
        <item>
         <layout class="QHBoxLayout" name="loConnect">
          <property name="topMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="icon">
             <iconset resource="../VfpRessources.qrc">
              <normaloff>:/root/Images/connect.png</normaloff>:/root/Images/connect.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDisconnect">
            <property name="text">
             <string>Disconnect</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Images/StopRed16.png</normaloff>../Images/StopRed16.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="loIPAddress">
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="lblIPAddress">
            <property name="text">
             <string>&amp;IP Address : </string>
            </property>
            <property name="buddy">
             <cstring>editIPAdress</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editIPAdress"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="loPort">
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="lblPort">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="buddy">
             <cstring>editPort</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editPort">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDirectCommands">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Direct Commands</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="loDirectCommands">
        <item>
         <widget class="QComboBox" name="cmbCommands">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
          <property name="icon">
           <iconset resource="../VfpRessources.qrc">
            <normaloff>:/root/Images/Send.png</normaloff>:/root/Images/Send.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnReceive">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Recieve</string>
          </property>
          <property name="icon">
           <iconset resource="../VfpRessources.qrc">
            <normaloff>:/root/Images/Receive.png</normaloff>:/root/Images/Receive.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbInstrumentsMessages">
     <property name="title">
      <string>Instrument Messages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="loInstrumentMessages">
        <item>
         <widget class="QTextEdit" name="editInstMsgs"/>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="icon">
           <iconset resource="../VfpRessources.qrc">
            <normaloff>:/root/Images/Clear32.png</normaloff>:/root/Images/Clear32.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnDisconnect</tabstop>
  <tabstop>editIPAdress</tabstop>
  <tabstop>editPort</tabstop>
  <tabstop>cmbCommands</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>btnReceive</tabstop>
  <tabstop>editInstMsgs</tabstop>
  <tabstop>btnClear</tabstop>
 </tabstops>
 <resources>
  <include location="../VfpRessources.qrc"/>
 </resources>
 <connections/>
</ui>
